src/Service/RequestHandler/Exception/InvalidRequestHandler.php

```diff
@@ -30,7 +32,7 @@ public function __construct(\Throwable $previous = null)
     {
         parent::__construct(
             'Invalid request handler object.',
-            1011,
+            Response::HTTP_INVALID_REQUEST_HANDLER,
```

Comment:
Isn't this a problem though? How do we handle this when translating to other carts?
For instance, in Shopware I just pickup the code from the thrown exception and assign it to the ```ShopgateResponse->setHttpCode()```. Do we just filter all 1000 codes into 500?

Reply:
Good point and I think that's what we should do. All of these SDK related errors come from misusage of the SDK when looking at the list of non-HTTP-codes. But I would still keep the different codes up until the end because it's easier to look for them, especially because the text can be changed when creating exceptions.